pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <Version>5.2.0</Version>
+    <PackagePlatform>AnyCPU</PackagePlatform>
+  </PropertyGroup>
+
+  <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+      <Output TaskParameter="Lines" ItemName="ProjectJsonLines" />
+    </ReadLinesFromFile>
+
+    <ItemGroup>
+      <PackageMatch Include="@(ProjectJsonLines)">
+        <PackageId>$([System.Text.RegularExpressions.Regex]::Match('%(Identity)', '(\w[^:&quot;]*)'))</PackageId>
+        <PackageVersion>$([System.Text.RegularExpressions.Regex]::Match('%(Identity)', '(\d+\.\d+.\d+[^&quot;]*)'))</PackageVersion>
+      </PackageMatch>
+
+      <Dependency Include="@(PackageMatch -> '%(PackageId)')" Condition="'%(PackageVersion)' != ''">
+        <Version>%(PackageVersion)</Version>
+      </Dependency>
+    </ItemGroup>
+
+    <PropertyGroup>
+      <VersionSuffix>-rc3-$(CLIBuildVersion)</VersionSuffix>
```

Comment:
Should this property be up in the general `PropertyGroup` next to <Version>?  It doesn't really make sense that it is inside this Target, since it has nothing to do with reading dependencies from project.json.